src/lib/fragments/Accordion.react.js

```diff
@@ -1,9 +1,16 @@
+import { transduce } from 'ramda';
```

Comment:
Looks like this isn't used

Reply:
Thanks Tom I'm not sure where this came from!